### ISSUE TITLE
Enhance game flow by allowing players to skip noobfilter section with new tile

### ISSUE THREAD

**Issue by murpii** [ddnet/ddnet#6518]
Noob filters remain a frequently encountered feature at the moment. This often requires players to repeatedly to do the noobfilter after a failed run, which can be frustrating and time-consuming on certain maps. I've personally experienced this just yesterday after playing through the entire Wonderfully map series. To improve this, I suggest introducing a new tile that allows players to bypass the noobfilter section after completing it successfully once.

**Comment by heinrich5991** [comment]
What would you propose to bind this to? Player name? (Account™? ^^) Session?

Perhaps we could require noob filters to not be so long or annoying on new maps? Making it possible to skip them might encourage mappers to make noob filters even longer and even more complicated.

**Comment by murpii** [comment]
> What would you propose to bind this to?
ClientID?
> Perhaps we could require noob filters to not be so long or annoying on new maps? 
I'd like to have this on older maps, not just new maps.
> Making it possible to skip them might encourage mappers to make noob filters even longer and even more complicated.
I don't think that'll be the case. Why?

**Comment by heinrich5991** [comment]
> > Making it possible to skip them might encourage mappers to make noob filters even longer and even more complicated.
> 
> I don't think that'll be the case. Why?

Because there are less incentives to keep it short if you don't have to re-do it on restart. An example where this happened is map sizes; after speeding up the map download, larger map sizes became tolerable and thus the size of maps increased.

**Comment by murpii** [comment]
> > > Making it possible to skip them might encourage mappers to make noob filters even longer and even more complicated.
> > 
> > 
> > I don't think that'll be the case. Why?
> 
> Because there are less incentives to keep it short if you don't have to re-do it on restart. An example where this happened is map sizes; after speeding up the map download, larger map sizes became tolerable and thus the size of maps increased.

I think this can be dealt with during the testing process if that would be the case.
- A noob-filter must not be too long or too hard. Its only purpose should be to slow down the arrival of new players, but not prevent them from playing at all.

**Comment by Chairn** [comment]
Im against that, noobfilters are part of the map and should not be skipped. Noob players might complete the nf once per chance and wouldn't have to do it anymore, which then often prevents t0 run to occur normally...

**Comment by edg-l** [comment]
I agree some noob filters are annoying, doing them 1 time per session should be enough IMHO.

Session being the duration between you connecting to the server and disconnecting.

**Comment by edg-l** [comment]
> Making it possible to skip them might encourage mappers to make noob filters even longer and even more complicated.

This can be avoided by testers or making a new rule to not make overlong noob filters, after all, we hold the say on what maps are added.

**Comment by murpii** [comment]
> Im against that, noobfilters are part of the map and should not be skipped. Noob players might complete the nf once per chance and wouldn't have to do it anymore, which then often prevents t0 run to occur normally...

I have never witnessed this before. Also I don't see how new players prevent t0 runs entirely? In fact, many people enjoy carrying new players in Team 0.

**Comment by AssassinTee** [comment]
Without accounts this can only cause abuse.
